packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5395,9 +5394,6 @@ export default class Meeting extends StatelessWebexPlugin {
 
     this.audio = createMuteState(AUDIO, this, audioEnabled);
     this.video = createMuteState(VIDEO, this, videoEnabled);
-
-    this.annotationInfo = localTracks?.annotationInfo;
```

Comment:
there is also the type declaration for `annotationInfo` around line 554

Reply:
remove it